ScoutSuite/core/conditions.py

```diff
@@ -181,6 +181,18 @@ def pass_condition(b, test, a):
             if re.match(c, b):
                 result = True
                 break
+    elif test == 'matchInList':
```

Comment:
Could you please add some tests to https://github.com/nccgroup/ScoutSuite/blob/master/tests/test_utils_conditions.py so codecov does not complain?